core/client/src/main/java/alluxio/client/block/BlockWorkerInfo.java

```diff
@@ -56,6 +56,13 @@ public long getCapacityBytes() {
   }
 
   /**
+   * @return the available bytes of the worker
+   */
+  public long getAvailableBytes() {
```

Comment:
@calvinjia  If use the strategy as you suggested, the reserved capacity is 5GB, is't suitable?